git/github.com/jenkins-x/jx3-pipeline-catalog.yml

```diff
@@ -1,30 +1,2 @@
 gitUrl: https://github.com/jenkins-x/jx3-pipeline-catalog.git
-version: 2f23f887114f0feb90d9c8b7c6a982c5625be89a
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+version: a936ed264d18158223d492c98e2d79a084ae6731
```

Comment:
Isn't it normally the sha of the PR? 
`c34ac730369e3139d52fdaf073e82cac74129f23`